### PR TITLE
added translations of phrases for the Lithuanian language

### PR DESCRIPTION
**Pull request by AndreyNegrov**
added translations of phrases for the Lithuanian language

### CODE REVIEW

File: locales/lt-LT/taoResultServer.rdf.po [hunk 1]
```diff
@@ -63,12 +63,12 @@ msgstr "Duomenys apie saugyklą yra išsiųsti į žurnalą"
 # http://www.tao.lu/Ontologies/TAOResultServer.rdf#implementation
 msgctxt "http://www.w3.org/2000/01/rdf-schema#label"
 msgid "Link to the implementation Class"
-msgstr ""
+msgstr "Link to the implementation Class"
```
Comment (bartlomiejmarszal) [inline locales/lt-LT/taoResultServer.rdf.po]: Why do you add English translation here?

File: locales/lt-LT/taoResultServer.rdf.po [hunk 2]
```diff
@@ -63,12 +63,12 @@ msgstr "Duomenys apie saugyklą yra išsiųsti į žurnalą"
...
 
 # http://www.tao.lu/Ontologies/TAOResultServer.rdf#implementation
 msgctxt "http://www.w3.org/2000/01/rdf-schema#comment"
 msgid "Link to the implementation Class"
-msgstr ""
+msgstr "Link to the implementation Class"
```
Comment (bartlomiejmarszal) [inline locales/lt-LT/taoResultServer.rdf.po]: Why do you add English translation here?

File: locales/lt-LT/taoResultServer.rdf.po (hunk 1 above)
Comment (AndreyNegrov) [reply locales/lt-LT/taoResultServer.rdf.po]: it was in the translation source files, I did not pay attention. It fixed

File: locales/lt-LT/taoResultServer.rdf.po (hunk 2 above)
Comment (AndreyNegrov) [reply locales/lt-LT/taoResultServer.rdf.po]: it was in the translation source files, I did not pay attention. It fixed

**Pull request closed without merging**